### PR TITLE
Update KC the WC property servletDestroyWaitTime

### PR DESCRIPTION
**Pull request by pmd1nh**
issue #3392

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -161,4 +161,7 @@ ignoreSemiColonOnRedirectToWelcomePage.name=Ignore semi-colon on redirect to wel
...
 
 useSemiColonAsDelimiterInURI.name=Use semi-colon as delimiter in URI
-useSemiColonAsDelimiterInURI.desc=Toggle to use the semi-colon as a delimiter in the request URI. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.useSemiColonAsDelimiterInURI.
\ No newline at end of file
+useSemiColonAsDelimiterInURI.desc=Toggle to use the semi-colon as a delimiter in the request URI. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.useSemiColonAsDelimiterInURI.
+
+servletDestroyWaitTime.name=Wait for an active request to complete
+servletDestroyWaitTime.desc=Wait time in second for an active request to complete when the owning application is stopped.  The default value is 60 sec.  The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.servletDestroyWaitTime.
```
Comment (sishida) [inline dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]: Wait time in **seconds** for an active request to complete when the owning application is stopped.  The default value is 60 **seconds**.

As stated by the IBM style guide, "Abbreviate units of time only when space is limited." So, if we're limited to a certain number of characters here, we can change it back to "sec." But if not, "seconds" should be spelled out. :)

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -161,4 +161,7 @@ ignoreSemiColonOnRedirectToWelcomePage.name=Ignore semi-colon on redirect to wel
 ignoreSemiColonOnRedirectToWelcomePage.desc=Toggle to ignore the trailing semi-colon when redirecting to the welcome page. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.ignoreSemiColonOnRedirectToWelcomePage.
 
 useSemiColonAsDelimiterInURI.name=Use semi-colon as delimiter in URI
-useSemiColonAsDelimiterInURI.desc=Toggle to use the semi-colon as a delimiter in the request URI. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.useSemiColonAsDelimiterInURI.
\ No newline at end of file
+useSemiColonAsDelimiterInURI.desc=Toggle to use the semi-colon as a delimiter in the request URI. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.useSemiColonAsDelimiterInURI.
```
[the remark this answers is not in the document]
Comment (sishida) [reply dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]: Also, sentences need only one space after periods.

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (sishida) [reply dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]: Also, sentences need only one space after periods.

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -161,4 +161,7 @@ ignoreSemiColonOnRedirectToWelcomePage.name=Ignore semi-colon on redirect to wel
...
+
+servletDestroyWaitTime.name=Wait for an active request to complete
```
[the remark this answers is not in the document]
Comment (sishida) [reply dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]: Can be left as-is if "Wait" is a verb.

**Comment by pmd1nh** [comment]
@sishida  I updated files.  Please review and approve.  Thanks!

**Comment by pmd1nh** [comment]
#build

**Comment by sishida** [comment]
Thank you for making the updates! I have added my "ID reviewed" label. :)

**Pull request merged**